test/controllers/EmailControllerSpec.scala

```diff
@@ -39,26 +47,50 @@ class EmailControllerSpec extends SpecBase {
     }
 
     "return unverified email response" in new Setup {
+
+      when[Future[EmailUnverifiedResponse]](
+        mockHttpClient.GET(ArgumentMatchers.endsWith("/subscriptions/unverified-email-display"), any, any)
+        (any, any, any))
+        .thenReturn(Future.successful(response))
+
       running(app) {
         val request = fakeRequest(GET, routes.EmailController.showUnverified().url)
         val result = route(app, request).value
         status(result) shouldBe OK
       }
     }
+
+    "return undeliverable email response" in {
```

Comment:
Could you please use Setup trait here in order to avoid duplication of creating app code and of httpClient?

Once done , you could remove below code as it will be come from the setup trait

 val mockHttpClient = mock[HttpClient]

 val app = applicationBuilder().overrides(
        bind[HttpClient].toInstance(mockHttpClient)
      ).build()

Reply:
fixed